modules/data.atmosphere/R/debias_met_regression.R

```diff
@@ -560,7 +569,11 @@ debias.met.regression <- function(train.data, source.data, n.ens, vars.debias=NU
         
         
         if(v == "precipitation_flux"){
-          Rbeta.ann <- matrix(MASS::mvrnorm(n=n.new, coef(mod.ann), vcov(mod.ann)), ncol=length(coef(mod.ann)))
+          if(n.ens==1){
```

Comment:
This is nit picky but we might want to catch the case of someone inputting n.ens=0 (as in "i don't want ensembles" instead of I want one ensemble member), which will have different responses in the ifelse statement n==1 and n>1 .